plugins/woocommerce-admin/client/products/use-product-helper.ts

```diff
@@ -70,8 +71,8 @@ export function useProductHelper() {
 	/**
 	 * Create product with status.
 	 *
-	 * @param {Product} product the product to be created.
-	 * @param {string}  status the product status.
+	 * @param {Product} product    the product to be created.
```

Comment:
Are these edits intended?

Reply:
The linter did this automatically.  Is this not the expected format?  If not, I can revert.